packages/flutter_markdown/test/markdown_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+const introductionText = '''
```

Comment:
Good teaser for our content! But how about making a much more minimalistic example that clearly show the potential problem:

```suggestion
'''
This is a paragraph containing both *italic* and **bold** spans and they should all render together without any line breaks.
'''
```